### PR TITLE
Query Images from Figma

### PR DESCRIPTION
**Pull request by emiliekmartinez**
This allows people to hit the Figma API and get back an image a node or many nodes!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ plugins: [
     options: {
       // For files:
       fileId: `FIGMA_FILE_ID`,
+      // For images:
+      nodeId: [`FIGMA_NODE_ID`],
```
Comment (fabe) [inline README.md]: As a developer, how would you get ahold of these IDs? How is this different from getting the `thumbnail` in the document's response?

Comment (fabe) [inline README.md]: This should be `nodeIds`.

File: src/gatsby-node.js [hunk 2]
```diff
@@ -21,12 +21,22 @@ See docs here – https://github.com/fabe/gatsby-source-figma
...
     const file = await Figma.fetchFile(fileId, accessToken);
+    
     files = [file];
-  } else if (projectId) {
+  } else if (fileId && nodeIds) {
+    const imagesData = nodeIds.map(nodeId =>
+      Figma.fetchNode(fileId, nodeId, scale, format, accessToken)
+    );
+
+    images = await Promise.all(imagesData);
```
Comment (fabe) [inline src/gatsby-node.js]: If `fileId` _and_ `nodeIds` are set, both should be available for querying. So we still need to make the `Figma.fetchFile` request that is currently skipped when both are set. Maybe try putting the image querying into a different if statement.

File: README.md [hunk 3]
```diff
@@ -80,7 +105,7 @@ Use the built-in GraphiQL tool (http://localhost:8000/___graphql) to get an idea
 
 * [x] Query `files`.
 * [ ] Query multiple `files`.
-* [ ] Query file `images`.
+* [x] Query file one or multiple `images`.
```
Comment (fabe) [inline README.md]: Should be:

> Query one or multiple file `images`.

**Review by fabe — COMMENTED** [review]
@emiliekmartinez Thanks for this! Left a couple comments.

File: README.md (hunk 1 above)
Comment (emiliekmartinez) [reply README.md]: Great question! I found the nodeIds in the URL of the Figma doc. You can think of it as "artboard ID". The thumbnail returned everything on the page, whereas this returns one particular artboard.

Should I add documentation around this?

File: src/gatsby-node.js (hunk 2 above)
Comment (emiliekmartinez) [reply src/gatsby-node.js]: Updated!

File: README.md (hunk 1 above)
Comment (fabe) [reply README.md]: Yes, adding a short sentence about this in the readme would be great!

File: README.md [hunk 4]
```diff
@@ -69,6 +69,11 @@ query StyleguideQuery {
 
 Make sure that `fileId`, `nodeId`, and `accessToken` are set inside `gatsby-config.js`. You can also set `scale` and `format`.
 
+
+`The node Id and file key can be parsed from any Figma node url: 
+(https://www.figma.com/file/:key/:title?node-id=:id).`
+
```
Comment (emiliekmartinez) [inline README.md:75]: @fabe Added it here!

**Comment by fabe** [comment]
Looks good, thank you!

**Pull request merged**